cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/SchemaService.java

```diff
@@ -83,32 +92,12 @@ public synchronized ApiResponse registerSchema(ClusterSchemaRequest clusterSchem
               null);
         }
       }
-      String suffixUrl =
-          clusterSchemaRequest.getEnv()
-              + "/subjects/"
-              + clusterSchemaRequest.getTopicName()
-              + "-value/versions";
-      Pair<String, RestTemplate> reqDetails =
-          clusterApiUtils.getRequestDetails(suffixUrl, clusterSchemaRequest.getProtocol());
-
-      Map<String, String> params = new HashMap<>();
-      params.put("schema", clusterSchemaRequest.getFullSchema());
-
-      HttpHeaders headers =
-          clusterApiUtils.createHeaders(
-              clusterSchemaRequest.getClusterIdentification(), KafkaClustersType.SCHEMA_REGISTRY);
-      headers.set("Content-Type", SCHEMA_REGISTRY_CONTENT_TYPE);
-      HttpEntity<Map<String, String>> request = new HttpEntity<>(params, headers);
-      ResponseEntity<String> responseNew =
-          reqDetails.getRight().postForEntity(reqDetails.getLeft(), request, String.class);
-
-      String updateTopicReqStatus = responseNew.getBody();
-      log.info("SchemaRequest response body {}", responseNew.getBody());
+      // register schema
+      String updateTopicReqStatus = registerSchemaPostCall(clusterSchemaRequest);
```

Comment:
This comment doesn't seem to add much to the codebase.

Reply:
Yea, removed.